pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotTableRestletResource.java

```diff
@@ -78,6 +78,17 @@ public Representation post(Representation entity) {
   @Tags({ "table" })
   @Paths({ "/tables", "/tables/" })
   private void addTable(AbstractTableConfig config) throws IOException {
+
+    // For self-serviced cluster, ensure that the tables are created with atleast
+    // min replication factor irrespective of table configuratio value.
+    int requestReplication = config.getValidationConfig().getReplicationNumber();
+    int configMinReplication = _controllerConf.getDefaultTableMinReplicas();
+    if (requestReplication < configMinReplication) {
```

Comment:
Can you please add the same check with 'replicasPerPartition' when creating llc tables?